react-konva.d.ts

```diff
@@ -40,16 +40,7 @@ export class KonvaContainerComponent<Container extends Konva.Container, Props =
   // mountAndAddChildren(): void;
 }
 
-export interface StageProps extends Pick<React.HTMLProps<any>, 'className' | 'role' | 'style' | 'tabIndex' | 'title'> {
-  x?: number;
-  y?: number;
-  scale?: {x: number, y: number};
-  scaleX?: number;
-  scaleY?: number;
-  name?: string;
-  width?: number | string;
```

Comment:
The only issue possibly worth noting is that the current implementation allows `number|string` for width/height.  however, konva api expects a number only.

https://github.com/konvajs/konva/blob/master/konva.d.ts#L72